operators/test/e2e/test/builder.go

```diff
@@ -20,3 +22,7 @@ type Builder interface {
 	// If the resource to mutate to is the same as the original resource, then all tests should still pass.
 	MutationTestSteps(k *K8sClient) StepList
 }
+
+func WithRandomPrefix(name string) string {
```

Comment:
Nit: Since the random string is appended to the name, this should probably be called  `WithRandomSuffix`

Reply:
:confused: